src/gui/issueswidget.cpp

```diff
@@ -104,6 +102,15 @@ IssuesWidget::IssuesWidget(QWidget *parent)
     _ui->_conflictHelp->setText(
         tr("There were conflicts. <a href=\"%1\">Check the documentation on how to resolve them.</a>")
             .arg(Theme::instance()->conflictHelpUrl()));
+
+    connect(FolderMan::instance(), &FolderMan::folderRemoved, this, [this](Folder *f) {
+        _model->remove_if([f](const ProtocolItem &item) {
+            if (item.folder() == f) {
```

Comment:
Can be a one line:
`return item.folder() == f`

Reply:
indeed